tox.ini

```diff
@@ -2,42 +2,49 @@
 skipsdist=True
 envlist=
     lint,
-    python{3.8,3.10}-django{3.2}-wagtail{2.15,3.0,4.0,4.1}
-    python{3.8,3.10}-django{4.0}-wagtail{3.0,4.0,4.1}
-    python{3.8,3.10}-django{4.1}-wagtail{4.0,4.1}
+    python{3.8}-django{3.2}-wagtail{3.0},
+    python{3.8,3.11}-django{3.2,4.1}-wagtail{4.1,4.2}
 
 [testenv]
 install_command=pip install -e ".[testing]" -U {opts} {packages}
 commands=
-    coverage erase
-    coverage run {toxinidir}/testmanage.py test {posargs}
-    coverage report --show-missing --fail-under=100
+    python -b -m coverage run --parallel-mode --source='wagtailinventory' {envbindir}/django-admin test {posargs}
```

Comment:
Ah, sorry, just synchronizing it with our other apps, which set `DJANGO_SETTINGS_MODULE` and use `django-admin`. 

I can change it back if you like... but I think it would be good to make it the same, if not now, then in another PR.

In truth, I just copied this line 😁 for the `--parallel-mode` for coverage combination in the `coverage` environment.

Reply:
I don't feel super strongly, but it's nice that testmanage.py sets the DJANGO_SETTINGS_MODULE so we don't have to do it separately. And then it's consistent with other uses of testmanage.py in the tox.ini -- those could theoretically use django-admin as well but having the separate script is convenient, and some other packages use like, like [wagtail-localize](https://github.com/wagtail/wagtail-localize/blob/main/testmanage.py).

I tend to think those advantages outweigh the inconsistency across our packages; if we find a better pattern it's nice to implement it where it can be helpful and port it over when a good opportunity presents itself.

But I will defer to you - my main issue was tox.ini having some use of django-admin and some use of the testmanage script.

Reply:
Yeah, I'll back that out here and we can modify it separately if we want. It just means being careful when copying iterations over from our other packages (unlike how I was here).

I kinda prefer the explicitness of setting the variable in the tox.ini, but it's such a minor thing.

Reply:
Reverted in [792848d](https://github.com/cfpb/wagtail-inventory/pull/67/commits/792848dd4c4f864c1214b311ea43ce1a80cffc09)